_data/alerts.yml

```diff
@@ -1,15 +0,0 @@
-tip: '<div class="alert alert-success" role="alert"><i class="fa fa-check-square-o"></i> <b>Tip: </b>'
```

Comment:
The `basic-sql-statements.md` and `start-a-local-cluster.md` files both reference this. Somehow, ever with this file removed, the callouts are still working. But for now, just to be safe, let's put this file back in.